src/oncall/ui/templates/index.html

```diff
@@ -517,6 +517,7 @@ <h4 class="modal-title">Calendar Timezones</h4>
           {{/isEqual}}
         </li>
         <li> Slack: <a>{{getUserInfo this @root.users 'contacts.slack'}}</a> </li>
+        <li> Pushover: <a>{{getUserInfo this @root.users 'contacts.pushover'}}</a> </li>
```

Comment:
most people will not be using pushover, it looks like this code will leave a bunch of empty  pushover related ui elements, these should be made conditional on pushover data existing.